seqr/management/commands/check_bam_cram_paths.py

```diff
@@ -31,13 +31,13 @@ def handle(self, *args, **options):
             individual__family__project__name__in=args
         ) if args else IgvSample.objects.all()).filter(
             file_path__startswith='gs://'
-        ).prefetch_related('individual', 'individual__family__project')
+        ).order_by('id').prefetch_related('individual', 'individual__family__project')
```

Comment:
Are there any reasons to add `order_by`?

Reply:
mostly test stability

Reply:
The test: https://github.com/broadinstitute/seqr/blob/a3cff8bc472d5b3efa6861e51e227b28f92f1453/seqr/management/tests/check_bam_cram_paths_tests.py#L43-L46
It uses a `sorted`. Does it stabilize the order?